src/models/Pet2.js

```diff
@@ -0,0 +1,83 @@
+const mongoose = require('mongoose');
+
```

Comment:
Hello! Just wondering what Pet2.js model is for.

Reply:
Hello! Just in case I left the old pet model (petModel2), we can delete it at any time. It is not used now, but in case if the new model will not be approved, we can delete it and use the old one.